### DISCUSSION TITLE
More Themes?

### DISCUSSION THREAD

**Discussion by crispybegs** [bpatrik/pigallery2#945] [Ideas]
Does there exist somewhere a collection of themes people have made? Like how Trilium and others have a [theme gallery](https://github.com/zadam/trilium/wiki/Theme-Gallery). I did search the discussions but it doesn't appear to have come up before.

![FireShot Capture 061 - Theme Gallery · zadam_trilium Wiki - github com](https://github.com/user-attachments/assets/31f05df6-0039-4a90-b6ae-2d0c8818f95b)

**Comment by bpatrik** [comment]
no not really. Feel free to create one and post it here to start the
process.

**Comment by crispybegs** [comment]
all good, thanks
